### ISSUE TITLE
JAWS and focus management issues

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/sonification-studio#17]
JAWS is not announcing elements as focus is set to them, tested with Chrome and Edge. Sometimes appears to announce them after a long time - could this be a performance issue related to the grid?

Examples:
- Help modal dialog when help button is clicked
- Helptexts when they are expanded
- The help buttons when helptexts are closed
- Open/save buttons when menus are collapsed using Esc

Chrome version: 91.0.4472.124
Edge version: 91.0.864.64
JAWS version: 2021.2105.53

**Comment by oysteinmoseng** [comment]
It seems the help dialog issue is due to JAWS just taking long to process, as the announcement eventually happens.

The helptexts are perhaps not announced because they are `<div>s` without a role. Not sure if we want to give them a role. `role="region"`or `role="complimentary"` could maybe be borderline acceptable?

**Comment by oysteinmoseng** [comment]
Redone helptexts now with a visible close button and title, to avoid potential issues with hidden buttons.

**Comment by oysteinmoseng** [comment]
The long process for JAWS on the help dialog seems to be only on the Data panel, and as such probably related to the grid and all of its ARIA grid DOM elements.

The collapsed menus on open/save still don't work as expected, but perhaps JAWS has its own handling of the Esc key.

The dialog role on helptexts seems to work pretty well.

Closing this for now.
